marklogic-unit-test-client/src/main/java/com/marklogic/test/unit/TestResult.java

```diff
@@ -8,11 +8,17 @@ public class TestResult {
 
 	private String name;
 	private double time;
+  private int tests;
+  private int success;
```

Comment:
"success" should be "succeeded" to match the tense of "failed" - but those sound like booleans. I think we need "count" as a suffix on all of these to make it clear what they're capturing. 

So `testCount`, `successCount`, and `failureCount`. They can also all be `final` as well, in addition to the existing 3 fields.